### DISCUSSION TITLE
Web copyright not updated

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#17] [Q&A] [answered]
![Screen Shot 2023-04-24 at 6 32 13 PM](https://user-images.githubusercontent.com/4314771/234152875-5820f67c-c3ad-4dac-9a97-a9bbe56ca579.png)
If you made changed in a specific year, i believe you ought to update it.

**Comment by azadbolour** [comment, marked as answer]
Thanks for the heads up. I will update the copyright years in the next
release.

Azad
